omod/src/main/webapp/resources/htmlforms/HMIS-MCH-008-INTEGRATED-POSTNATAL-REGISTER.xml

```diff
@@ -0,0 +1,629 @@
+<htmlform formName="HMIS MCH 008: INTEGRATED POSTNATAL REGISTER"
+          formDescription="Encounter form for patients enrolled for PNC services"
+          formEncounterType="fa6f3ff5-b784-43fb-ab35-a08ab7dbf074"
+          formUuid="a6c5a8c3-c414-449f-a077-c2f6acbdd8b1"
+          formVersion="1.2"
+          formAddMetadata="yes"
+          formUILocation="patientDashboard.visitActions"
+          formOrder="6"
+          formIcon="icon-medkit"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false"
+          formDisplayStyle="Standard"
+          formLabel="Postnatal Visit Form">
+          <!-- &definitionUiResource=ugandaemrpoc:htmlforms/test.xml -->
```

Comment:
@shamim2019 This is not correct - fix the name of the file being used

Reply:
done